jgrapht-core/src/main/java/org/jgrapht/intervalgraph/interval/Interval.java

```diff
@@ -118,21 +133,87 @@ public int compareTo(Interval<T> o) {
     }
 
     @Override
-    public String toString() {
+    public String toString()
+    {
         return "Interval[" + start + ", " + end + "]";
     }
 
     @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+    public boolean equals(Object o)
+    {
+        if (this == o)
```

Comment:
well, I think we should set up a formatter for Eclipse / IntelliJ to avoid reformatting between different commits.

Reply:
probably yes. I just auto formatted the whole class, since I did some style flaws and just assumed the rest of the class was also auto formatted.